src/firestore/fsConfig.ts

```diff
@@ -31,9 +31,23 @@ export function getFirestoreConfig(projectId: string, options: Options): ParsedF
     }
   }
 
+  // If user specifies firestore:rules or firestore:indexes make sure we don't throw an error if this doesn't match a database name
+  if (onlyDatabases.has("rules")) {
+    onlyDatabases.delete("rules");
+    allDatabases = true;
+  }
+  if (onlyDatabases.has("indexes")) {
+    onlyDatabases.delete("indexes");
+    allDatabases = true;
+  }
+
   // single DB
   if (!Array.isArray(fsConfig)) {
-    if (fsConfig) {
+    if (onlyDatabases) {
+      throw new FirebaseError(
+        "Please use multiple database firebase.json format to specify database names in --only option. Refer to https://firebase.google.com/docs/cli#firestore-multiple-dbs for more details"
```

Comment:
Will that be feasible to add some unittests to cover the journey and error message?